pennylane/labs/tests/phase_polynomials/test_phase_polynomial.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for pennylane/labs/phase_polynomials/phase_polynomial.py"""
+import numpy as np
+import pytest
+
+import pennylane as qml
+from pennylane.labs.phase_polynomials import phase_polynomial
+
+# example from fig. 1 in https://arxiv.org/abs/2104.00934
+circ1 = qml.tape.QuantumScript(
+    [
+        qml.CNOT((1, 0)),
+        qml.RZ(1, 0),
+        qml.CNOT((2, 0)),
+        qml.RZ(2, 0),
+        qml.CNOT((0, 1)),
+        qml.CNOT((3, 1)),
+        qml.RZ(3, 1),
+    ],
+    [],
+)
+
```

Comment:
I think I would suggest adding one or two more examples, maybe including an empty circuit?

Reply:
empty circuit is a good idea!
Not sure about more examples, what kind of cases would you consider?

Reply:
Not sure, actually :sweat_smile: I thought maybe a larger circuit, but the logic of the function is simple enough to test it on a small-scale circuit, I suppose :) So no need to add more, I think :+1: